hbase-common/src/main/java/org/apache/hadoop/hbase/TableName.java

```diff
@@ -83,6 +83,12 @@
   public static final TableName META_TABLE_NAME =
       valueOf(NamespaceDescriptor.SYSTEM_NAMESPACE_NAME_STR, "meta");
 
+  /** hbase:slowlog table name - can be enabled
+   * with config - hbase.regionserver.slowlog.systable.enabled
+   */
+  public static final TableName SLOW_LOG_TABLE_NAME =
```

Comment:
We are exposing this Table name for users right? I think we need.  Then we should expose the cf and column names also?  Sure this is not the class for those expose.
Now we allow all kind of client ops (writes, disable, alter) on this table? I believe in Master branch we allow tables in system NS to be disabled, altered etc.  There was a Jira around META table.  Will that be ok (Disable this table and keep it that state)

Reply:
Yes, I tried disable and enable of 'hbase:slowlog' locally and both worked fine.